test/acceptance/features/steps/steps.py

```diff
@@ -13,13 +13,10 @@
 
 from behave import given, register_type, then, when, step
 from dboperator import DbOperator
```

Comment:
@baijum This should be also removed along with the class and step:
see the https://github.com/baijum/service-binding-operator/pull/2

Reply:
Thanks for the PR. I have merged it.